code/_onclick/hud/hud.dm

```diff
@@ -57,6 +57,7 @@ GLOBAL_LIST_INIT(available_ui_styles, list(
 	var/obj/screen/healths
 	var/obj/screen/healthdoll
 	var/obj/screen/internals
+	var/obj/screen/wanted_lvl

```

Comment:
If you make this static you can remove the for loop in the gamemode's process and replace it with something a lot simpler. That comes with some issues though so it's your call.

Reply:
see below msg about it